hpx/util/annotated_function.hpp

```diff
@@ -114,7 +114,8 @@ namespace hpx { namespace util
 #endif
 
     ///////////////////////////////////////////////////////////////////////////
-    namespace detail
+    /// \brief Gets useful information for the input function.
+		namespace detail
```

Comment:
Could you please check the indentation in your changes? There are tabs in front of `namespace detail`, while we prefer to use spaces everywhere.